README.md

```diff
@@ -260,22 +260,22 @@ build:
         current: String | List of Strings
         upstream: String | List of Strings
         downstream: String | List of Strings
-        merge: String[]
       current: String | List of Strings
       upstream: String | List of Strings
       downstream: String | List of Strings
       after:
         current: String | List of Strings
         upstream: String | List of Strings
         downstream: String | List of Strings
-        merge: String[]
-      merge: String[]
-    skip: true
     archive-artifacts:
       name: String
       path*: String | List of Strings
       if-no-files-found (default:warn): warn|ignore|error
       dependencies (default:none): all|none|list of project names
+    clone: List of Strings
+
+  - project: ANOTHER_PROJECT_NAME
+    skip: true
```

Comment:
we are trying to provide project json structure here, do you think a new entry should be added instead of keeping `skip` as a part of the previous one? The idea is to provide information to the developer about the JSON pproject fields. I guess what you are trying to suggest here is `skip` doesn't make sense with any additional fiel, but... wdyt?

Reply:
I added a new entry to show that you cannot have `skip: true` if you have `build-command` or `archive-artifacts`

Reply:
is that use case reflected as a test scenario?

- just skip -> OK
- skip + build-command and/or archive-artifacts -> Exception

Reply:
There are too many schema combinations possible so I didn't write tests for all of them. I only wrote tests for the more important schema combinations just as a sanity check (this not being one of them).

I can go ahead and write test cases for this as well

Reply:
Nvm for backporting purposes looks like `skip` will have to be allowed along with `build-command` or `archive-artifacts`

https://github.com/kiegroup/build-chain-configuration-reader/blob/ab0a664fe7a6c7fb95ec2903eb817629be4796fb/test/resources/build-config-with-dependencies-for-example.yaml#L215

Reply:
i will update the readme